packages/block-editor/src/components/block-draggable/draggable-chip.js

```diff
@@ -13,7 +13,10 @@ import BlockIcon from '../block-icon';
 export default function BlockDraggableChip( { count, icon } ) {
 	return (
 		<div className="block-editor-block-draggable-chip-wrapper">
-			<div className="block-editor-block-draggable-chip">
+			<div
+				className="block-editor-block-draggable-chip"
+				data-testid="block-draggable-chip"
```

Comment:
We shouldn't add mark up just for the test. Is that a leftover?

Reply:
No, it's intentional. `data-testid` is not ideal but might be the best we can do for these UI-only elements. It's an established pattern used by most modern testing libraries. See this [blog post](https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change#whats-with-the-data-testid-query) for more info.

Reply:
Thanks for sharing. Okay, it seems no big deal, especially since it's on the editor.